code/src/components/app-table/components/app-table-body/AppTableBody.types.ts

```diff
@@ -0,0 +1,3 @@
+import { TableBodyProps } from "@mui/material/TableBody";
+
+export type AppTableBodyProps = TableBodyProps;
```

Comment:
Where do you used this type ?
just import this `TableBodyProps` in file. 
Don't create type which you pass existing type from `MUI`

Reply:
Because it's our code convention, do not use type directly from `Mui`. Look our components folder